Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -300,10 +306,18 @@ import BraintreeDataCollector
         
         authenticationSession?.presentationContextProvider = self
         returnedToAppAfterPermissionAlert = false
+        apiClient.sendAnalyticsEvent(BTPayPalAnalytics.browserPresentationStarted)
         isAuthenticationSessionStarted = authenticationSession?.start() ?? false
 
         let authenticationSessionStatus = isAuthenticationSessionStarted ? "succeeded" : "failed"
```

Comment:
We also don't need this string value anymore. We can just use the `isAuthenticationSessionStarted` bool directly in the `if` statement below.

Reply:
do you mean the authenticationSessionStatus string? or that we don't need the browserPresentationStarted event?